content/templates/_index.md

```diff
@@ -26,15 +26,11 @@ weight: 2
 <p>No results were found for your search. Try adjusting your search.</p>
 </div>
 
-<div id="results">
-  <h2 style="padding-bottom: 20px">Boilerplates</h2>
-  {{< boilerplates >}}
+{{<gallery>}}
 
-  <h2 style="padding-bottom: 20px">Snippets</h2>
-  {{< snippets >}}
-
-  <h2 style="padding-bottom: 20px">Featured Projects</h2>
-  {{< featured_boilerplates >}}
+<br>
```

Comment:
can we add some padding to the above div to space this out instead of using `<br />`?